### ISSUE TITLE
New users have nothing to do while waiting on feedback

### ISSUE THREAD

**Issue by dpritchett** [exercism/exercism.io#475]
Exercism is very engaging, but users whose first Bob has not yet been approved have nothing to do and limited insight into how the whole thing works.
#### Ways forward
1. Give people some very clear explanations of what regular interactions look like for people who've finished Bob.
2. Come up with something to do for folks who haven't yet received a single approval.

This is all rather `nice-to-have` but I do worry that frustrations in the onboarding process could turn away some potentially desirable community members.

Note: I haven't seen this state in a week or two so I can't really say if the onboarding has changed materially since I went through it.  Comments on Parley suggest that people are still confused when they are in this first-exercise limbo.
